docs/index.md

```diff
@@ -1,4 +1,10 @@
+<<<<<<< before updating
 # **ess**sans
+||||||| last update
+# Esssans
+=======
+# ESSsans
+>>>>>>> after updating
```

Comment:
conflict

Reply:
Ah, I was wondering why the diff was rendered strangely.